extensions/devservices/oidc/src/main/java/io/quarkus/devservices/oidc/OidcDevServicesProcessor.java

```diff
@@ -797,17 +811,19 @@ private static void userInfo(RoutingContext rc) {
             String token = authorization.substring("Bearer ".length());
             JsonObject claims = decodeJwtContent(token);
             if (claims != null && claims.containsKey(Claims.preferred_username.name())) {
+                String sub = claims.getString(Claims.sub.name());
+                String name = claims.getString("name") == null ? sub : claims.getString("name");
```

Comment:
It this `name` claim actually set somewhere during the token generation ?

Reply:
The `name` claim is always set in `createAccessToken`. I removed this additional check here therefore.